package.json

```diff
@@ -7,10 +7,10 @@
     "npm": ">=3.8"
   },
   "scripts": {
-    "start": "node tools/run.js",
-    "build": "node tools/build.js",
-    "build:debug": "node tools/build.js --debug",
-    "build:lib": "node tools/lib.build.js && flow-copy-source -i '**/@(__mocks__|__tests__)/**' ./src ./dist && mv ./dist/lib.js.flow ./dist/index.js.flow",
+    "start": "babel-node tools/run.js",
```

Comment:
babel-node should not be used in production: https://babeljs.io/docs/en/babel-node
 As this is only used in development this should be fine. I'd rather avoid it though if possible in any way.

Reply:
Yeah, I wanted to avoid it as well, but then the flow-typed AppConfig was (for me) an argument for using it...